src/init/mod.rs

```diff
@@ -279,4 +287,25 @@ mod tests {
         assert_eq!(starship_path.sprint_pwsh()?, r"'C:\''starship.exe'");
         Ok(())
     }
+
+    #[test]
+    fn escape_cmd() -> io::Result<()> {
```

Comment:
Similar name changes here (`escape_cmd` sounds like we're testing escaping inputs to an external command).